src/components/collection_vote_box/CollectionSubmitVoteBox.jsx

```diff
@@ -106,13 +107,18 @@ export default function CollectionSubmitVoteBox(props) {
 
   const updateVote = async (newVote, isChair) => {
     try {
-      const voteIds = map(v => v.voteId)(votes);
+      const openElectionVotes = votes.filter(v => toLower(v.electionStatus) === 'open');
+      const voteIds = map(v => v.voteId)(openElectionVotes);
```

Comment:
```suggestion
      const voteIds = openElectionVotes.map(v => v.voteId);
```

Avoids use of `lodash/fp` , which is no longer being updated and causes typing issues with Typescript.

Reply:
Gotcha! Agreed, directly using the map is better.